rct229/rulesets/ashrae9012019/section19/section19rule10.py

```diff
@@ -0,0 +1,271 @@
+            index_rmr="baseline",
+            id="19-10",
+            description="Air economizers shall be included in baseline HVAC Systems 3 through 8, and 11, 12, and 13 based on climate as specified in Section G3.1.2.6 with exceptions."
+            "1. Systems that include gas-phase air cleaning to meet the requirements of Standard 62.1, Section 6.1.2. This exception shall be used only if the system in the proposed design does not match the building design."
+            "2. Where the use of outdoor air for cooling will affect supermarket open refrigerated case-work systems. This exception shall only be used if the system in the proposed design does not use an economizer. If the exception is used, an economizer shall not be included in the baseline building design."
+            "3. Systems that serve computer rooms complying with Section G3.1.2.6.1.",
+            ruleset_section_title="HVAC - General",
+            standard_section="Section G3.1.2.6 including exceptions 1-3",
+            is_primary_rule=True,
+            list_path="$.ruleset_model_instances[*].buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmr_b = context.baseline
+
+        assert_(
+            rmr_b["ruleset_model_instances"],
+            "Please make sure `ruleset_model_instances` exists in the `baseline`.",
+        )
+        rmi_b = rmr_b["ruleset_model_instances"]
+
+        climate_zone = rmr_b["weather"]["climate_zone"]
+
+        baseline_system_types_dict_b = get_baseline_system_types(rmi_b)
+
+        return (climate_zone not in NOT_APPLICABLE_CLIMATE_ZONE) and any(
+            [
+                baseline_system_type_compare(system_type, applicable_sys_type, False)
+                for system_type in baseline_system_types_dict_b.keys()
+                for applicable_sys_type in APPLICABLE_SYS_TYPES
+            ]
+        )
+
+    def create_data(self, context, data):
+        rmr_b = context.baseline
+        rmi_b = rmr_b["ruleset_model_instances"][0]
+
+        rmr_p = context.proposed
+        assert_(
+            rmr_p["ruleset_model_instances"],
+            "Please make sure `ruleset_model_instances` exists in the `proposed`.",
+        )
+        rmi_p = rmr_p["ruleset_model_instances"][0]
+
+        proposed_has_economizer = is_economizer_modeled_in_proposed(rmi_b, rmi_p)
+
+        hvac_system_exception_2_list = []
+        if rmi_b.get("refrigerated_cases") is not None and not proposed_has_economizer:
+            hvac_system_exception_2_list = [
+                hvac_id_b
+                for hvac_id_b in find_all(
+                    f'$.buildings[*].building_segments[*][?(@.lighting_building_area_type = "{LIGHTING_BUILDING_AREA.RETAIL}")].heating_ventilating_air_conditioning_systems[*].id',
+                    rmi_b,
+                )
+            ]
+
+        baseline_system_types_dict = get_baseline_system_types(rmi_b)
```

Comment:
This can move to the `create_data` in RMI level class.

Reply:
Addressed.